flang/test/Driver/func-attr.f90

```diff
@@ -6,7 +6,7 @@
 subroutine func
 end subroutine func
 
-! CHECK-NONEFP: attributes #0 = { "frame-pointer"="none" }
+! CHECK-NONEFP: attributes #0 = {{{.*}}}
```

Comment:
Instead of `{{.*}}` could you check that there are no attributes here? This way we check that no frame pointer attributes are added for `None`. `{{.*}}` can match anything.